### ISSUE TITLE
Create message payload from template files

### ISSUE THREAD

**Issue by acogoluegnes** [rabbitmq/rabbitmq-perf-test#224]
From [the mailing list](https://groups.google.com/d/msg/rabbitmq-users/HzSA28Ns97Y/P1Y6G77YAgAJ).

Payload for published messages could be based on one or several template files that would contain placeholders to make the content dynamic, e.g.:

```
{
    "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9",
    "payload": {
        "trxid": "DEPOSIT-PUBSUB-${uuid}",
        "trx_amount": ${list('5000','10000','50000')},
        "trx_notes": "${list('Deposit','Expense')}"
    }
}
```

Other candidate "functions" in placeholder could be `sequence`, `random` (not an UUID, just a random, 10-character long string).

Functions could accept arguments, e.g. `${random(length:16, cache:50)}`.

**Comment by typekpb** [comment]
any timeline on this one?
For me would be the increment function important

**Comment by michaelklishin** [comment]
@typekpb this is open source software so you are welcome to contribute it.

**Comment by michaelklishin** [comment]
@acogoluegnes is there a library that handles such template functions or we'd have to effectively develop a tiny templating language ourselves? In the latter case this probably shouldn't be labeled with `effort-low` :)

**Comment by acogoluegnes** [comment]
@michaelklishin Fair enough, this will require a custom development, adding `effort-medium` :-)
